module/wallpaper.lua

```diff
@@ -310,7 +330,11 @@ function setup(args)
     config.set_function = config.set_function
         or (config.wallpaper and setters.simple or setters.awesome_wallpaper)
     local function set_wallpaper(s)
-        config.screen = s or config.screen
+        if type(config.screen) == 'table' then
+        else
```

Comment:
Just do `~=` here instead of an empty `if` block

Reply:
Done, remnant of the old code